api/userCrud/userRouter.js

```diff
@@ -0,0 +1,64 @@
+const express = require('express');
+const userModel = require('./userModel');
+const router = express.Router();
+const authRequired = require('../middleware/authRequired');
+
+
+router.get('/:company_id/users', authRequired, function (req, res) {
+    userModel.getCompanyUsers(req.params.company_id)
+      .then((users) => {
+        res.status(200).json(users);
+      })
+      .catch((err) => {
+        console.log(err);
+        res.status(500).json({ message: "Something went wrong", err });
+      });
+  });
+
+router.get('/:company_id/user/:user_id', function (req, res) {
+  userModel.getCompanyUser(req.params.company_id, req.params.user_id)
+        .then((user) => {
+          res.status(200).json(user);
+        })
+        .catch((err) => {
+          console.log(err);
+          res.status(500).json({ message: "user not found", err });
+        });
+})
+
+router.post('/:company_id/user', function (req, res) {
```

Comment:
"Either that or we might want to go back into the migrations soon and make sure the primary key of all our tables, id, auto increments so we don't need to worry about passing in id's ourself."
About the id increments, I thought that it will be passed in using an id we get back from okta.

Reply:
Like this id string "00ulthapbErVUwVJy4x6"...